NorthwindCRUD/Providers/DbContextConfigurationProvider.cs

```diff
@@ -92,15 +92,15 @@ private MemoryCacheEntryOptions GetCacheConnectionEntryOptions()
                 AbsoluteExpirationRelativeToNow = TimeSpan.FromHours(defaultAbsoluteCacheExpirationInHours),
             };
 
-            cacheEntryOptions.RegisterPostEvictionCallback(CloseConnection);
+            cacheEntryOptions.RegisterPostEvictionCallback(CloseConnection!);
```

Comment:
Does `CloseConnection` need non-null assertion `!` if it's a static method?

Reply:
Wow. So the exclamation mark was not for the method itself, but for the parameters inside it. I didn't know that was possible.